File: app/src/main/java/com/coppermobile/mysampleusingmvvmlivedata/data/Comments.java
package com.coppermobile.mysampleusingmvvmlivedata.data;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.support.annotation.NonNull;

@Entity(tableName = "comments_table")
public class Comments {

    @NonNull
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "comment_id")
    private int id;

    @ColumnInfo(name = "comment_name")
    private String comments;

    @ColumnInfo(name = "comment_time")
    private String date;

    @ColumnInfo(name = "dish_id")
    private int dishID;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getComments() {
        return comments;
    }

    public void setComments(String comments) {
        this.comments = comments;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public int getDishID() {
        return dishID;
    }

    public void setDishID(int dishID) {
        this.dishID = dishID;
    }
}

File: app/src/main/java/com/coppermobile/kotlinsamplemvvmlivedataroom/data/Comments.kt
package com.coppermobile.kotlinsamplemvvmlivedataroom.data

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "comments_table")
class Comments {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "comment_id")
    var id: Int = 0

    @ColumnInfo(name = "comment_name")
    var comments: String? = null

    @ColumnInfo(name = "comment_time")
    var date: String? = null

    @ColumnInfo(name = "dish_id")
    var dishId: Int = 0
}